include/ignition/gui/resources.qrc

```diff
@@ -1,21 +1,21 @@
-<!DOCTYPE RCC><RCC version="1.0">
-<qresource>
-  <file>qtquickcontrols2.conf</file>
-
-  <file>qml/IgnCard.qml</file>
-  <file>qml/IgnCardSettings.qml</file>
-  <file>qml/IgnHelpers.qml</file>
-  <file>qml/IgnRulers.qml</file>
-  <file>qml/IgnSpinBox.qml</file>
-  <file>qml/IgnSplit.qml</file>
-  <file>qml/Main.qml</file>
-  <file>qml/PluginMenu.qml</file>
-  <file>qml/SideDrawer.qml</file>
-  <file>qml/StandaloneDialog.qml</file>
-  <file>qml/StyleDialog.qml</file>
-
-  <file>qml/images/ignition_logo_50x50.png</file>
-  <file>qml/images/drawer.png</file>
-  <file>qml/images/menu.png</file>
-</qresource>
+<RCC>
+    <qresource prefix="/">
+        <file>qtquickcontrols2.conf</file>
+        <file>qml/IgnCard.qml</file>
+        <file>qml/IgnCardSettings.qml</file>
+        <file>qml/IgnHelpers.qml</file>
+        <file>qml/IgnRulers.qml</file>
+        <file>qml/IgnSpinBox.qml</file>
+        <file>qml/IgnSplit.qml</file>
+        <file>qml/Main.qml</file>
+        <file>qml/PlottingInterface.qml</file>
+        <file>qml/PluginMenu.qml</file>
+        <file>qml/SideDrawer.qml</file>
+        <file>qml/StandaloneDialog.qml</file>
+        <file>qml/StyleDialog.qml</file>
+        <file>qml/images/ignition_logo_50x50.png</file>
+        <file>qml/images/drawer.png</file>
+        <file>qml/images/menu.png</file>
+        <file>qml/Chart.qml</file>
+    </qresource>
```

Comment:
Are all these indentation changes necessary?

Reply:
No :D 
i just add one line in the middle and all these changes are added
they don't have any changes except one line

Reply:
Does your editor have some setting to convert spaces into tabs or something like that? :monocle_face: I can help you revert these.